src/main/java/frc/robot/subsystems/CoralElevator.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class CoralElevator extends SubsystemBase {
+
+    private DoubleSolenoid piston1;
+    private DoubleSolenoid piston2;
+    private DigitalInput UpperLimitSwitch;
+    private DigitalInput LowerLimitSwitch;
+
+    public CoralElevator() {
+        piston1 = new DoubleSolenoid(PneumaticsModuleType.REVPH, RobotMap.PISTON_FORWARD_CHANNEL, RobotMap.PISTON_REVERSE_CHANNEL);
+        piston2 = new DoubleSolenoid(PneumaticsModuleType.REVPH, RobotMap.PISTON_FORWARD_CHANNEL, RobotMap.PISTON_REVERSE_CHANNEL);
+        UpperLimitSwitch = new DigitalInput(1);
```

Comment:
```suggestion
        UpperLimitSwitch = new DigitalInput(RobotMap.CORAL_ELEVATOR_UPPER_LIMIT_SWITCH);
```
Use ports from `RobotMap`